ERROR 2022-07-15 22:05:09,767 - Error Message from get_list_item
Traceback (most recent call last):
  File "d:\Tinghao.Chen\Desktop\oneTouch_tool\test_plan_ui.py", line 687, in mainWork
    if self.checkBox.isChecked():
AttributeError: 'runBatchcommand' object has no attribute 'checkBox'
ERROR 2022-07-15 22:05:13,705 - Error Message from get_list_item
Traceback (most recent call last):
  File "d:\Tinghao.Chen\Desktop\oneTouch_tool\test_plan_ui.py", line 687, in mainWork
    if self.checkBox.isChecked():
AttributeError: 'runBatchcommand' object has no attribute 'checkBox'
